### ISSUE TITLE
Documentation reg. admin login not very clear

### ISSUE THREAD

**Issue by muelli** [debops/ansible-gitlab#16]
Currently, it reads "The credentials are saved in the secrets dir secret/credentials/host/gitlab/admin/password The login is admin@host".

That information, however, is not entirely accurate.  The only user configured for gitlab seems to be the user with the login `gitlab_admin_email` which defaults to `user@{{ ansible_domain }}`. That information would be good to have.  It would probably also be good to save that username along with the password in the secret folder or print it out during provisioning.

**Comment by drybjed** [comment]
Sure, might be good idea to store the username in secrets, however that would need to be set by the user in inventory. I'll look what can be done. Also, `root` admin account works the same as the admin email.

**Comment by drybjed** [comment]
@muelli I don't want to use `debug` module in the roles directly, this breaks confidentiality of the GitLab passwords stored in secrets. IMO this would be better done by emailing the administrator about how to find the password in DebOps secrets, without mailing the password itself.

**Comment by muelli** [comment]
fair enough.  No hard feelings at all.  Just providing what seems to work well enough for me.
